server/src/immich/controllers/person.controller.ts

```diff
@@ -26,8 +29,16 @@ export class PersonController {
   constructor(private service: PersonService) {}
 
   @Get()
-  getAllPeople(@AuthUser() authUser: AuthUserDto): Promise<PersonResponseDto[]> {
-    return this.service.getAll(authUser);
+  getAllPeople(
+    @AuthUser() authUser: AuthUserDto,
+    @Query(new ValidationPipe({ transform: true })) withHidden: PersonSearchDto,
+  ): Promise<PeopleResponseDto> {
+    return this.service.getAll(authUser, withHidden);
+  }
+
+  @Get('statistics')
+  getPersonStats(@AuthUser() authUser: AuthUserDto): Promise<PersonCountResponseDto> {
+    return this.service.getPersonCount(authUser);
```

Comment:
We decided to remove this method, right?

Reply:
I said It could be interesting for those who use the statistics endpoint, but it's clearly not vital

Reply:
I guess I'm ok either way. I think we can have the stats endpoint and keep get all people as it originally was (return a list of people) or remove stats and add the totals into the get all people. I don't think we should do both though.